Python/bytecodes.c

```diff
@@ -2839,11 +2831,9 @@ dummy_func(
         // (Some args may be keywords, see KW_NAMES, which sets 'kwnames'.)
         // On exit, the stack is [result].
         // When calling Python, inline the call using DISPATCH_INLINED().
-        inst(CALL, (unused/1, unused/2, method, callable, args[oparg] -- res)) {
-            int is_meth = method != NULL;
+        inst(CALL, (unused/1, unused/2, callable, self_or_null, args[oparg] -- res)) {
             int total_args = oparg;
-            if (is_meth) {
-                callable = method;
+            if (self_or_null) {
```

Comment:
This is perfectly correct, but could you make this consistent with the other NULL checks and use `self_or_null != NULL` to make it clear to the reader that this is a `NULL` check not a test of a boolean/int flag.